docs(FAQ): question about the second node in rdb ext-add-faq_rdb (#1089)
scaleway/docs-content: faq/databases-for-postgresql-and-mysql.mdx

|-----------------	|-----------------------	|----------------	|------------------------------	|-----------------------------	|------------------	|
| 192.168.0.0/24  	| 255.255.255.0         	| 24             	| 192.168.0.1                  	| 192.168.0.254               	| 192.168.0.33/24  	|
| 10.18.0.0/16    	| 255.255.0.0           	| 16             	| 10.18.0.1                    	| 10.18.255.254               	| 10.18.21.42/16   	|
| 192.168.18.0/26 	| 255.255.255.192       	| 26             	| 192.168.18.1                 	| 192.168.18.63               	| 192.168.18.37/26 	|

## Can I use the second node on High Available Managed Database Instance ?

> No, the second node created when you activate the HA on your Managed Database Instance is a hot standby. This node allow fast failover in case of issue on the Primary Node.

Reviewer: ```suggestion
No, the second node created when you activate the HA on your Managed Database Instance is a hot standby. This node allows a fast failover in case of issues on the primary node.
```

Author: Solved

> No, the second node created when you activate the HA on your Managed Database Instance is a hot standby. This node allow allows a fast failover in case of issues on the primary node.

Reviewer: ```suggestion
No, the second node created when you activate the HA on your Managed Database Instance is a hot standby. This node allows a fast failover in case of issues on the primary node.
```

> Please use [read replica feature](/docs/managed-databases/postgresql-and-mysql/how-to/create-read-replica/).

Reviewer 2: ```suggestion
Please use [read replica feature](/managed-databases/postgresql-and-mysql/how-to/create-read-replica/).
```
